src/main/java/nextstep/member/ui/FavoriteController.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.member.ui;
+
+import nextstep.member.application.FavoriteService;
+import nextstep.member.application.config.AuthFavorite;
+import nextstep.member.application.dto.FavoriteRequest;
+import nextstep.member.application.dto.FavoriteResponse;
+import nextstep.member.domain.Member;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+public class FavoriteController {
+    private final FavoriteService favoriteService;
+
+    public FavoriteController(FavoriteService favoriteService) {
+        this.favoriteService = favoriteService;
+    }
+
+    @PostMapping("/favorites")
+    public ResponseEntity<Void> createFavorite(@AuthFavorite Member member, @RequestBody FavoriteRequest param) {
+        favoriteService.createFavorite(member, param);
+        return ResponseEntity.created(URI.create("/favorites/")).build();
+    }
```

Comment:
생성한 엔티티의 URL을 응답해야하지 않을까요?
```suggestion
    @PostMapping("/favorites")
    public ResponseEntity<Void> createFavorite(@AuthFavorite Member member, @RequestBody FavoriteRequest param) {
        var favorite = favoriteService.createFavorite(member, param);
        return ResponseEntity.created(URI.create("/favorites/" + favorite.getId())).build();
    }
```